openehr-rm/src/main/java/com/nedap/archie/rm/changecontrol/VersionedObject.java

```diff
@@ -42,11 +42,11 @@ public void setOwnerId(ObjectRef ownerId) {
         this.ownerId = ownerId;
     }
 
-    public DvDateTime getTimeCreations() {
+    public DvDateTime getTimeCreated() {
```

Comment:
TODO: maybe postpone to major release?

Reply:
This one is just plain inconsistent with the field name and serializations, so even though it is backwards incompatible, I'll change it.